package/Dockerfile

```diff
@@ -44,39 +37,21 @@ RUN wget -q -O - "${ETCD_URL}" | tar xzf - -C /tmp && \
 COPY templates /etc/confd/templates/
 COPY conf.d /etc/confd/conf.d/
 COPY cert-deployer nginx-proxy /usr/bin/
-COPY entrypoint.sh cloud-provider.sh weave-plugins-cni.sh /opt/rke-tools/
+COPY entrypoint.sh cloud-provider.sh /opt/rke-tools/
 COPY rke-etcd-backup /opt/rke-tools
 
 VOLUME /opt/rke-tools
 CMD ["/bin/bash"]
 
 # Temporary image mostly to verify all binaries exist and are
 # valid for the target architecture.
-FROM tonistiigi/xx:1.2.1 AS xx
+FROM tonistiigi/xx:1.4.0 AS xx
 FROM base as test
 COPY --from=xx / /
 
 ARG TARGETOS=linux
 ARG TARGETARCH=${ARCH}
 
-RUN xx-verify --static /tmp/bandwidth \
-    && xx-verify --static /tmp/bridge \
-    && xx-verify --static /tmp/dhcp \
-    && xx-verify --static /tmp/firewall \
-    && xx-verify --static /tmp/flannel \
-    && xx-verify --static /tmp/host-device \
-    && xx-verify --static /tmp/host-local \
-    && xx-verify --static /tmp/ipvlan \
-    && xx-verify --static /tmp/loopback \
-    && xx-verify --static /tmp/macvlan \
-    && xx-verify --static /tmp/portmap \
-    && xx-verify --static /tmp/ptp \
-    && xx-verify --static /tmp/sbr \
-    && xx-verify --static /tmp/static \
-    && xx-verify --static /tmp/tuning \
-    && xx-verify --static /tmp/vlan \
-    && xx-verify --static /tmp/vrf
```

Comment:
Aren't these binaries used for the other CNI plugins (e.g. flannel needs the bridge CNI plugin)? They might come from another place but just in case I ask

Reply:
In fact, I created this issue when I was looking at RKE1 but I never had time: https://github.com/rancher/rke/issues/3327